registry/accessing-the-registry.adoc

```diff
@@ -32,13 +32,15 @@ the user must have the `registry-viewer` role. To add this role:
 $ oc policy add-role-to-user registry-viewer <user_name>
 ----
 
-* For writing or pushing images, for example when using the `podman push` command,
-the user must have the `registry-editor` role. To add this role:
+* For writing or pushing images, for example when using the `podman push` command:
+** The user must have the `registry-editor` role. To add this role:
 +
 [source,terminal]
 ----
 $ oc policy add-role-to-user registry-editor <user_name>
 ----
++
+** You must have an existing registry project.
```

Comment:
You must have an "existing project", not an "existing registry project". I would rephrase that to make things clear:

** You can only push to a project that already exists in your cluster.

Reply:
Thanks @ricardomaraschini . I have made the suggested changes.